src/components/Chart.jsx

```diff
@@ -36,7 +36,7 @@ class Chart extends Component {
     }
 
     render() {
-        const { DrawToolsSettingsDialog, StudySettingsDialog, children, lang, isMobile } = this.props;
+        const { DrawToolsSettingsDialog, StudySettingsDialog, children, lang, isMobile, isShowChartUnavailable } = this.props;
```

Comment:
nitpick: rename to `isStreamingAllowed` or `isChartAvailable`.

Reply:
I changed it to `isChartAvailable`